pandas/tests/io/test_html.py

```diff
@@ -765,11 +1054,11 @@ def test_keep_default_na(self):
                     </table>"""
 
         expected_df = DataFrame({'a': ['N/A', 'NA']})
-        html_df = read_html(html_data, keep_default_na=False)[0]
+        html_df = self.read_html(html_data, keep_default_na=False)[0]
```

Comment:
What was the point of changing some of these?

Reply:
These tests are all run twice: once with `lxml` and once with `bs4`. But `read_html()`'s logic is, "try `lxml`, and if it fails use `bs4`." So it overcomes the entire purpose of the unit tests -- if the lxml backend _crashes_ while we're testing it, we won't know because `read_html()` will fall back to `bs4`.

From my reading of the code, `self.read_html()` is always correct, and `read_html()` is always a bug.

Reply:
OK thanks for the clarification. I suppose these are fine to leave as is though if there are more better to do in a separate change to keep that cleanup orthogonal to the original issue